plugins.rst

```diff
@@ -22,11 +22,15 @@ Overview
 
 The following is the list of official plugins by |omv|.
 
-* **ClamAV**: Provides antivirus scan for folders.
+* **ClamAV**: Provides Clam AntiVirus (ClamAV). It is a free software, cross-platform and open-source antivirus software toolkit able to detect many types of malicious software, including viruses
+* **Diskstats**: Complementary plugin to extend system statistics collection by adding iostats graphs.
```

Comment:
Could you please change that to the following. I think this phrasing is more clear than the original (moved) text.
```suggestion
* **Diskstats**: Complementary plugin to extend system statistics collection by adding I/O statistic graphs.
```